packages/frontend/src/components/Settings/Settings.tsx

```diff
@@ -34,6 +34,19 @@ export default function Settings({ onClose }: DialogProps) {
   const tx = useTranslationFunction()
   const [settingsMode, setSettingsMode] = useState<SettingsView>('main')
 
+  useEffect(() => {
+    document.onkeydown = (evt: KeyboardEvent) => {
+      if (
```

Comment:
Of course it is not ideal to use the generic document.onkeydown here but it works and solves the issue. Once we need another document.onkeydown we have to look for a better solution